src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
@housseindjirdeh well, I rolled back and look in the master branch, this problem is there.

Remind me what I need to do? Just now PR is not clickable, and fix the title associated with it is better (in my opinion) in PR, which will do it (it will be more understandable)